client/src/components/Applications/AppForm/AppForm.js

```diff
@@ -185,10 +190,15 @@ export const AppSchemaForm = ({ app }) => {
         state.allocations.hosts[defaultHost] ||
         hasCorral,
       defaultStorageHost: defaultHost,
-      hasStorageSystems: configuration.length
+      hasStorageSystems: configuration.length,
+      downSystems: state.systemMonitor
+        ? state.systemMonitor.list
+            .filter(currSystem => !currSystem.is_operational)
+            .map(downSys => downSys.hostname)
+        : [],
+      execSystem: state.app ? state.app.exec_sys.login.host : ''
     };
   }, shallowEqual);
-
```

Comment:
No need to change this now, but in the future let's try to avoid no-op changes like this, unless it was changed for a particular reason

Reply:
Will resolve this, accidentally remove blank lines all the time while console logging.